### ISSUE TITLE
How should client apps properly mute log message emitted by your package?

### ISSUE THREAD

**Issue by atc0005** [dasrick/go-teams-notify#4]
I'm fairly new to Go still, so there is probably an obvious way to handle this, but is there an easy way to mute log messages generated by your package when errors are encountered?

For example:

https://github.com/dasrick/go-teams-notify/blob/feea812b67f873b4959ec21f2f2c5265126e13e2/send.go#L49-L54

Here the message is both logged and returned to the caller. From my app I've got the error message that was returned, but I also get the log output. 

To work around that I've used an approach like this to temporarily mute all log output:

```golang
    log.SetOutput(ioutil.Discard)
    if err := sendMessage(webhook.WebhookURL, msgCard); err != nil {
        // our error handling here
    }
    log.SetOutput(os.Stdout)
```

Is this the recommended approach, or is there a way to more directly target log messages generated by this package?

Thanks in advance for your time.

**Comment by dasrick** [comment]
You are right. This package should only return an error instead of doing both (logging and returning).
I will take it into one of the next changes ...

**Comment by atc0005** [comment]
Awesome, thank you.

I did some digging last night to see if Go handles this in the same way as Python does and got back mixed information. One example I did find common to both was setting up a package level (exported) logger instance/object and setting its output to disabled by default. The package user could then opt to enable the package-level logger if they need to see relevant troubleshooting/verbose details from the package. Not sure if that's overkill for this package or if it would be "freeing" in a sense to allow for more potential logging. 

I think I might be rambling at this point, so I'll leave off here.
